src/artifacts.js

```diff
@@ -0,0 +1,58 @@
+const
```

Comment:
This is just a personal habit. I just fall in the first camp here. http://eslint.org/docs/rules/one-var

I'm more than happy to paint that shed whatever color you like :)